ansible/roles/girder-histomicstk/tasks/main.yml

```diff
@@ -1,61 +1,43 @@
 ---
 - name: NodeJS | Add PPA key
   command: "apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 68576280"
-  become: true
+  become: yes
+  become_user: root
 
 - name: NodeJS | Add PPA
   apt_repository:
-    repo: "deb https://deb.nodesource.com/node_6.x {{ ansible_distribution_release }} main"
-  become: true
+    repo: "deb https://deb.nodesource.com/node_4.x {{ ansible_distribution_release }} main"
+  become: yes
+  become_user: root
 
 - name: NodeJS | Install package
   apt:
     name: nodejs
-    install_recommends: no
```

Comment:
I suggest we should always use `install_recommends: no` when installing apt packages.  It (a) makes smaller docker images, and (b) makes our actual requirements much more explicit.  In the past, some of the suggested packages have changed resulting in not installing something we've actually needed.